config/v1/types_infrastructure.go

```diff
@@ -531,15 +531,15 @@ type AWSResourceTag struct {
 	// key is the key of the tag
```

Comment:
Looks like the documentation of this field is a bit sparse. While you are making changes here, it would be good to update the godoc here to explain a bit more to a user how this field is used and what the constraints are. Users aren't able to see the same markers we are, so a good description here goes a long way in making the user experience of using `oc/kubectl explain` to learn more about the APIs they are interacting with better.

A good guideline for some things to consider when writing godoc for a field can be found here: https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#write-user-readable-documentation-in-godoc